src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/ComputeManagementClient.json

```diff
@@ -3521,6 +3521,10 @@
           "$ref": "#/definitions/VirtualMachineInstanceView",
           "readOnly": true,
           "description": "Gets the virtual machine instance view."
+        },
```

Comment:
@hyonholee @stankovski 
Did you also update: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-compute/2015-06-15/swagger/compute.json ?